### PR TITLE
Adding HTTP fetcher

### PR DESCRIPTION
**Pull request by dbemiller**
For people who want to fetch Stored Requests through an API.

### CODE REVIEW

File: pbs_light.go [hunk 1]
```diff
@@ -961,6 +962,13 @@ func NewFetchers(cfg *config.StoredRequests, db *sql.DB) (byId stored_requests.F
 		byAmpId = db_fetcher.NewFetcher(db, cfg.Postgres.MakeAmpQuery)
 		ampIdList = append(ampIdList, byAmpId)
 	}
+	if cfg.HTTP != nil {
+		glog.Infof("Loading Stored Requests from Postgres. endpoint=%s, amp_endpoint=%s", cfg.HTTP.Endpoint, cfg.HTTP.AmpEndpoint)
```
Comment (cirla) [inline pbs_light.go]: s/Postgres/HTTP

Comment (dbemiller) [reply pbs_light.go]: derp... thanks

File: stored_requests/backends/http_fetcher/fetcher.go [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+	}
+}
+
+func unpackResponse(resp *http.Response) (requestData map[string]json.RawMessage, impData map[string]json.RawMessage, errs []error) {
+	respBytes, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		errs = append(errs, err)
+		return
+	}
+	defer resp.Body.Close()
```
Comment (hhhjort) [inline stored_requests/backends/http_fetcher/fetcher.go]: So we do not need to close if there was an error in reading the resp.Body?

Comment (dbemiller) [reply stored_requests/backends/http_fetcher/fetcher.go]: mmm.... no, I think that's a bug. Is fixed now.

**Pull request merged**